packages/components/src/checkbox/docs/CheckboxGroup.stories.tsx

```diff
@@ -0,0 +1,207 @@
+import { Label } from "../../Label/index.ts";
+import { Inline } from "../../layout/index.ts";
+import { Text } from "../../Text/index.ts";
+import { Checkbox } from "../src/Checkbox.tsx";
+import { CheckboxField } from "../src/CheckboxField.tsx";
+import { CheckboxGroup } from "../src/CheckboxGroup.tsx";
+import { CheckboxList } from "../src/CheckboxList.tsx";
+
+/**
+ * The CheckboxGroup is used to group related options together and to provide a label for the group. It can also provide validation and helper messages.
+ *
+ * [View repository](https://github.com/gsoft-inc/wl-hopper/tree/main/packages/components/src/Checkbox/src)
+ * -
+ * [View package](https://www.npmjs.com/package/@hopper-ui/components)
+ * -
+ * View storybook TODO
+ */
+const meta = {
+    title: "Docs/Checkbox/CheckboxGroup",
+    tags: ["autodocs"],
+    parameters: {
+        // Disables Chromatic's snapshotting on documentation stories
+        chromatic: { disableSnapshot: true }
+    },
+    component: CheckboxGroup
+} satisfies Meta<typeof CheckboxGroup>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+/**
+ * A default CheckboxGroup.
+ */
+export const Default = {
+    render: props => (
+        <CheckboxGroup {...props}>
+            <CheckboxList>
+                <Checkbox value="developer">Developer</Checkbox>
+                <Checkbox value="designer">Designer</Checkbox>
+                <CheckboxField>
+                    <Checkbox value="manager">Manager</Checkbox>
+                    <Text slot="description">Team Manager</Text>
+                </CheckboxField>
+            </CheckboxList>
+        </CheckboxGroup>
+    ),
+    args: {
+        defaultValue: ["developer"],
+        "aria-label": "Roles"
+    }
+} satisfies Story;
+
+/**
+ * A CheckboxGroup can have a Label to provide more context to the user.
+ */
+export const LabelStory = {
+    name: "Label",
+    render: props => (
+        <CheckboxGroup {...props}>
+            <CheckboxList>
+                <Checkbox value="developer">Developer</Checkbox>
+                <Checkbox value="designer">Designer</Checkbox>
+                <Checkbox value="manager">Manager</Checkbox>
+            </CheckboxList>
+            <Label>Roles</Label>
+        </CheckboxGroup>
+    )
+} satisfies Story;
+
+/**
+ * A CheckboxGroup can have a description to provide more information to the user.
+ */
+export const Description = {
+    render: props => (
+        <CheckboxGroup {...props}>
+            <Label>Roles</Label>
+            <CheckboxList>
+                <Checkbox value="developer">Developer</Checkbox>
+                <Checkbox value="designer">Designer</Checkbox>
+                <CheckboxField>
+                    <Checkbox value="manager">Manager</Checkbox>
+                    <Text slot="description">Team Manager</Text>
+                </CheckboxField>
+            </CheckboxList>
+            <HelperMessage hideIcon>Select one to continue</HelperMessage>
```

Comment:
i remember Francis B saying that all helper message and error message inside the CheckboxGroup should hide their icon by default. This props should be passed why the CheckboxGroup using the HelperMessageContext. This way consumer don't have to add this props everytime